testutil-mentions/src/main/kotlin/io/spine/examples/pingh/testing/mentions/given/PredefinedGitHubResponses.kt

```diff
@@ -69,13 +70,21 @@ public class PredefinedGitHubResponses : GitHubClientService {
      */
     private var responseStatusCode = HttpStatusCode.OK
 
+    /**
+     * List of values of the `updatedAfter` arguments which were passed to [fetchMentions] methods.
+     *
+     * The order of the elements corresponds to the order of methods call.
```

Comment:
This is strange. I don't get the idea of storing something separately from the objects they relate to.

Let's discuss your original intention vocally.